packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -107,7 +108,12 @@ export const DeclarativeChart: React.FunctionComponent<DeclarativeChartProps> =
   if (!chart.isValid) {
     throw new Error(`Invalid chart schema: ${chart.errorMessage}`);
   }
-  const plotlyInput = plotlySchema as PlotlySchema;
+  let plotlyInput = plotlySchema as PlotlySchema;
+  try {
+    plotlyInput = decodeBase64Fields(plotlyInput);
```

Comment:
add this same check in plotlySchemaConverter in chart-utilities also

Reply:
done